apps/site/assets/ts/helpers/departureInfo.ts

```diff
@@ -107,9 +107,21 @@ const mergeIntoDepartureInfo = (
   return sortedDepartureInfo;
 };
 
+const isAtDestination = (
+  stopName: string,
+  route: Route,
+  directionId: DirectionId
+): boolean => {
+  return (
+    stopName ===
+    route.direction_destinations[directionId]!.replace(" Station", "")
```

Comment:
suggestion: some direction destinations might be named "Stop 1 or Stop 2", so I think `route.direction_destinations[directionId].includes(stopName)` will be more flexible and work ok here. though I do feel like a more robust solution might be needed eventually.

Reply:
I didn't do includes since some headsigns have `or` . For example 450 headsign could be "Wonderland or Haymarket" and didn't want to drop incase its going to Haymarket. Open to suggestions for this

Reply:
ohh yeah I see what you're saying, that's a good point. 
though I'm not sure you'll actually hit that condition given how the logic leading to this call is structured, because when you're checking this in the departure rows you're in the not condition of `Object.keys(groupedDepartures).length > 0` (which is derived from departures filtered for this direction)... so if I'm interpreting this right you're already in the situation where there are 0 departures for the entire direction anyway.